core/src/main/java/com/arcrobotics/ftclib/util/FTCLibOpMode.java

```diff
@@ -0,0 +1,45 @@
+package com.arcrobotics.ftclib.util;
+
+import com.arcrobotics.ftclib.gamepad.GamepadEx;
+import com.qualcomm.hardware.lynx.LynxModule;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+
+import java.util.List;
+
+public abstract class FTCLibOpMode extends LinearOpMode {
+
+    protected GamepadEx driverOp, toolOp;
+
+    @Override
+    public final void runOpMode() throws InterruptedException {
+        driverOp = new GamepadEx(gamepad1);
+        toolOp = new GamepadEx(gamepad2);
```

Comment:
Assuming the driver is using `gamepad1` and the operator is using `gamepad2` is not a good idea. I also dislike the names `driverOp` and `toolOp`; I would never think that gamepads have the word `op` in them, and `toolOp` especially doesn't make sense to me. I'd go for `gamepadEx1` and `gamepadEx2` or something similar.

Reply:
Fair enough. This is just convention I picked up from working with FRC.